htdocs/comm/propal/card.php

```diff
@@ -959,7 +959,7 @@
 		$remise_percent = (GETPOST('remiseforalllines') ? GETPOST('remiseforalllines') : 0);
 		$remise_percent = str_replace('*', '', $remise_percent);
 		foreach ($object->lines as $line) {
-			$result = $object->updateline($line->id, $line->subprice, $line->qty, $remise_percent, $line->tva_tx, $line->localtax1_tx, $line->localtax2_tx, $line->desc, 'HT', $line->info_bits, $line->special_code, $line->fk_parent_line, 0, $line->fk_fournprice, $line->pa_ht, $line->label, $line->product_type, $line->date_start, $line->date_end, $line->array_options, $line->fk_unit, $line->multicurrency_subprice);
+			$result = $object->updateline($line->id, $line->subprice, $line->qty, $remise_percent, $line->tva_tx ." ($line->vat_src_code)", $line->localtax1_tx, $line->localtax2_tx, $line->desc, 'HT', $line->info_bits, $line->special_code, $line->fk_parent_line, 0, $line->fk_fournprice, $line->pa_ht, $line->label, $line->product_type, $line->date_start, $line->date_end, $line->array_options, $line->fk_unit, $line->multicurrency_subprice);
```

Comment:
If line->vat_src_code is not defined, the concat of " (vatcode)" must not be done.

Reply:
I did it like this in next pr:
```php
   $tvatx= $line->tva_tx;
    if ( !empty($line->vat_src_code) && !preg_match('/\(/', $tvatx)) {
        $tvatx .= '('.$line->vat_src_code.')';
    }
   $result = $object->updateline($line->id, $line->subprice, $line->qty, $remise_percent, $tvatx, ....)
```